.github/workflows/auto_updates.yml

```diff
@@ -0,0 +1,58 @@
+# Workflow for updating Javascript dependencies with npm.
+# Major version updates are handled separately, by Dependabot.
+---
+name: Update Deps
+
+on:
+  workflow_dispatch:
+  # Run every Monday at 1435 UTC
+  schedule:
+    - cron: '35 14 * * 1'
+
+jobs:
+  workflow-auto-updates:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Set up Node
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
+        with:
+          node-version: '*'
+          cache: 'npm'
+          cache-dependency-path: site/package-lock.json
+
+      - name: Update Docusaurus Javascript dependencies
+        working-directory: ./site
+        run: npm update --save
+
+      - name: Commit changes
+        id: commit
+        continue-on-error: true
+        run: |
+          git config user.name 'phylum-bot'
+          git config user.email '69485888+phylum-bot@users.noreply.github.com'
+          git commit -a -m "build: bump static site Javascript dependencies"
+          git push --force origin HEAD:workflow-auto-updates
+
+      - name: Create Pull Request
+        if: ${{ steps.commit.outcome == 'success' }}
+        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          # This PAT is for the `phylum-bot` account and only has the `public_repo` scope to limit privileges.
```

Comment:
This is not the place to document the privileges of our access tokens. First, this repo is public and everyone can see this. Second, this could easily change in the future.